cmd/meroxa/flink/deploy.go

```diff
@@ -64,6 +69,31 @@ func GetIRSpec(ctx context.Context, jarPath string, secrets map[string]string, l
 	}
 
 	spec.Secrets = secrets
+	// workarounds to validate spec
+	spec.Definition.Metadata.SpecVersion = ir.LatestSpecVersion
+	spec.Definition.Metadata.Turbine.Language = "js" // java and flink are not acceptable yet
+	spec.Definition.Metadata.Turbine.Version = majorJavaVersion
+
+	// hardcode all sources to one destination as streams
+	destinationUUID := ""
+	var sourceUUIDs []string
+	for _, cs := range spec.Connectors {
+		if cs.Type == ir.ConnectorDestination {
+			destinationUUID = cs.UUID
+		} else {
+			sourceUUIDs = append(sourceUUIDs, cs.UUID)
+		}
+	}
+
+	for _, u := range sourceUUIDs {
```

Comment:
This limits us to fanning in only, correct? I'm assuming that this will be revisited in a later epic.

Reply:
oh yeah, definitely... this is a hack just to unblock us, but we don't actually have insight into the real DAG for any Flink Job at this point. every join and transformation would change which input eventually goes to which output. this is also a problem for when we try to figure out the output topics in the state machine in platform API.  if we have the dag, we can find the upstream entity, but without it, we're having to keep track of that relationship outside of the IR spec.